src/mwocp68.ron

```diff
@@ -88,5 +97,87 @@
                 }),
             ),
         },
+
+        "MFR_PAGE_X": {
```

Comment:
Where did this come from?  I don't see it in the "PSU Firmware Update Process" doc or ACAN-114.